### ISSUE TITLE
Battle actions involving an item cause a crash

### ISSUE THREAD

**Issue by Anonymous941** [Herringway/earthbound#121]
When I try to attack or use a good, the game crashes with this error:

```plain
core.exception.ArrayIndexError@source/earthbound/bank02.d(3539): index [18446744073709551615] is out of bounds for array of length 32
```

**Comment by Herringway** [comment]
Possibly fixed by 858a43c6. can you confirm?

**Comment by Anonymous941** [comment]
Yes, it works now
